app/lib/pdf_filler/az140_pdf.rb

```diff
@@ -83,6 +83,9 @@ def hash_for_pdf
         "12" => @xml_document.at('FedAdjGrossIncome')&.text,
         "14" => @xml_document.at('ModFedAdjGrossInc')&.text,
         "30" => @xml_document.at('USSSRailRoadBnft')&.text,
+        "31" => @xml_document.at('WageAmIndian')&.text,
+        "32" => @xml_document.at('CompNtnlGrdArmdFrcs')&.text,
+        "35" => @xml_document.at('TotalSubtractions')&.text,
```

Comment:
this was already being calculated but never added to xml or pdf. it's a calculation based on the new fields for this ticket so i went ahead and added it

Reply:
line 35 that is